src/views/static/css/scams.css

```
@@ -252,4 +245,62 @@ thead th {
     display: flex;
     justify-content: center;
     align-items: center;
+}
+
+#results-table,.search-grid-frame,.scams-zero-entries,.scams-no-query-results,.next-page {
+  display: none;
+}
+
+.scams-zero-entries,.scams-no-query-results {
+  text-align: center;
+  font-weight: bold;
+  margin-bottom: 100px;
+}
+
+.active {
+	color: white;
+	font-weight: bold;
+	background-color: #ffd166;
```

Why was this changed from red?

```suggestion
	background-color: red;
```

Makes the page look funky now.

---

Also, why is it square?